git lab install - issue on wait gitlab

Exits: 

```bash
panic: failed to execute kubectl wait for gitlab pods with label app=webservice:
exit status 1

goroutine 1 [running]:
log.Panicf({0x1c8d409?, 0x21?}, {0xc000bed5e0?, 0xa?, 0xc0006c4c00?})
	/usr/local/go/src/log/log.go:392 +0x67
github.com/kubefirst/kubefirst/internal/k8s.WaitForGitlab(0xd8?, 0xc0003b3080)
	/home/developer/app/internal/k8s/kubernetes.go:301 +0x2db
github.com/kubefirst/kubefirst/cmd.glob..func16(0xc0001f0000?, {0xc000014ad0, 0x0, 0x1})
	/home/developer/app/cmd/createGitlab.go:211 +0x12c8
github.com/kubefirst/kubefirst/cmd.glob..func13(0x2be1300?, {0xc000014ad0, 0x0, 0x1})
	/home/developer/app/cmd/create.go:90 +0x43f
github.com/spf13/cobra.(*Command).execute(0x2be1300, {0xc000014ac0, 0x1, 0x1})
	/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:856 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0x2be1580)
	/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
github.com/kubefirst/kubefirst/cmd.Execute()
	/home/developer/app/cmd/root.go:32 +0x25
main.main() 
```

## Logs: 

```bash 
LOG: 2022/10/10 11:46:53.453740 /home/developer/app/pkg/shell.go:25: ERR:
LOG: 2022/10/10 11:46:53.453983 /home/developer/app/cmd/createUtils.go:253: gitlab namespace found, continuing
LOG: 2022/10/10 11:46:59.055977 /home/developer/app/pkg/shell.go:23: Command Execution: /home/developer/.k1/tools/kubectl
LOG: 2022/10/10 11:46:59.056348 /home/developer/app/pkg/shell.go:24: OUT:
LOG: 2022/10/10 11:46:59.056615 /home/developer/app/pkg/shell.go:25: ERR: No resources found in gitlab namespace.

LOG: 2022/10/10 11:46:59.056855 /home/developer/app/cmd/createUtils.go:265: gitlab pods found, continuing
LOG: 2022/10/10 11:47:14.058128 /home/developer/app/cmd/createGitlab.go:210: waiting for gitlab
LOG: 2022/10/10 11:47:14.830863 /home/developer/app/internal/k8s/kubernetes.go:301: failed to execute kubectl wait for gitlab pods with label app=webservice:
exit status 1
```

--- a/internal/flagset/installer.go
+++ b/internal/flagset/installer.go
@@ -143,8 +143,10 @@ func ProcessInstallerGenericFlags(cmd *cobra.Command) (InstallerGenericFlags, er
 		log.Println("Error processing addons:", err)
 		return InstallerGenericFlags{}, err
 	}
-	viper.Set("addons", addonsFlag)
-	log.Println("addons", addonsFlag)
+	addons := viper.GetStringSlice("addons")
+	addons = append(addons, addonsFlag...)
+	viper.Set("addons", addons)
+	log.Println("addons", addons)
 
 	experimentalMode, err := ReadConfigBool(cmd, "experimental-mode")
 	if err != nil {

